content/articles/choosing-wagtail.md

```diff
@@ -0,0 +1,404 @@
+            <td class="text-center">
+            <i class="fas fa-lg fa-check-circle" aria-label="yes" style="color: #0c0;"></i>
+            </td>
+            <td class="text-center">
+            <i class="fas fa-lg fa-check-circle" aria-label="yes" style="color: #0c0;"></i>
+            </td>
+            <td class="text-center">
+            <span style="color: #999;"><small>No need</small></span>
+            </td>
+            <td class="text-center">
+            <span style="color: #999;"><small>No need</small></span>
+            </td>
+            <td class="text-center">
+            <span style="color: #999;"><small>No need</small></span>
+            </td>
+        </tr>
+    </tbody>
+    <tfoot>
+        <tr>
+            <td colspan="7">
+              <i class="fas fa-sm fa-check-circle" aria-label="yes" style="color: #0c0;"></i> =  <small>Yes</small> | 
+              <i class="fas fa-sm fa-circle" aria-label="Somewhat" style="color: #999;"></i> =  <small>Somewhat</small> | 
+              <i class="far fa-sm fa-circle" aria-label="No" style="color: #999;"></i> = <small>No</small>
+              
+            </td>
+        </tr>
+    </tfoot>
+</table>
+
+Django CMS and Wagtail are the two CMS frameworks that met all of our combined
+requirements. So, we compared how each against another set of developer-centric
+criteria:
+
+* Can the CMS __exist with other applications__ in a Django project?
+* How __flexible__ is the framework? Can the data model, and the CMS model be
+__extended__?
+* How __simple__ is it to implement, and to use?
+* Is the CMS UI device agnostic (__responsive__)?
+* Can we integrate a search engine?
+* What is the extend of __support and documentation__? How active is the
+__community__ around the CMS framework?
+* How easy is it to __learn to use__ the CMS framework?
+
+<table class="table table-bordered">
+    <thead>
+        <tr>
+            <th style="width: 20%;">
+            Criteria
+            </th>
+            <th class="text-center" style="width: 40%;">
+            <img src="/img/assets/thumb-djcms.png" style="width: 150px" alt="Django CMS" title="Django CMS" aria-label="Django CMS" />
+            </th>
+            <th class="text-center" style="width: 40%;">
+            <img src="/img/assets/thumb-wagtail.png" style="width: 150px" alt="Wagtail" title="Wagtail" aria-label="Wagtail" />
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>
+            <b>Multiple applications in one project</b>
+            </td>
+            <td>
+            <small>CMS UI uses Django admin code, can’t separate admin and CMS UI. More of a standalone product.</small>
+            </td>
+            <td>
+            <i class="fas fa-lg fa-check-circle" aria-label="yes" style="color: #0c0;"></i> <small>Non-invasive Django Wagtail app. Admin for Wagtail is separate from Django admin. Can still manage applications code in the same project without using Wagtail CMS.</small>
```

Comment:
I think "applications" should be singular to match code

Reply:
Fixed